bundle/manifests/vertical-pod-autoscaler.clusterserviceversion.yaml

```diff
@@ -759,6 +740,42 @@ spec:
           - create
           - patch
         serviceAccountName: vertical-pod-autoscaler-operator
+      - rules:
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - create
+        - apiGroups:
+          - coordination.k8s.io
+          resourceNames:
+          - vpa-recommender
+          resources:
+          - leases
+          verbs:
+          - get
+          - watch
+          - update
+        serviceAccountName: vpa-recommender
+      - rules:
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - create
+        - apiGroups:
+          - coordination.k8s.io
+          resourceNames:
+          - vpa-updater
+          resources:
+          - leases
+          verbs:
+          - get
+          - watch
+          - update
+        serviceAccountName: vpa-updater
```

Comment:
Do you know where these rules came from?

Reply:
Yes, that reveals a process problem in this repo. They came from #180, but I didn't run `make bundle` on that PR. We should probably have a CI step which runs `make bundle` and compares to the current bundle to the CI-generated one to make sure there are no real differences and catches me when I forget to run it.

Reply:
I guess it's from here but we didn't do `make bundle` somewhere. https://github.com/openshift/vertical-pod-autoscaler-operator/blob/8e649b20074cb06f0d3839c27f2be9507d6f1011/config/vpa/vpa-rbac.yaml#L370